media/system/js/fields/fielduser.js

```diff
@@ -93,10 +93,10 @@ Joomla = window.Joomla || {};
 
 	Joomla.fieldUser = function(options){
 		return options.forEach(function(){
-			var el = event.target, instance = el.getAttribute('fieldUser');
+			var el = event.target, instance = el.getAttribute('data-fieldUser');
 			if(!instance){
 				var options = options || {},
-					data = el.getAttribute();
+					data = el.getAttribute('data');
```

Comment:
```
data = el.getAttribute('data');
```
that will return null, you need to define the attributes and then go through with some thing like `for (var p in attribs) {}`

I have already shared a snippet to get you started...

Reply:
Anyways please don't spend your time here as this and the media field.js are already converted in the main repo

Reply:
Yeah sure. Anyway, I'll update these issues. Thanks for reviewing :)